<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBarClass</class>
 <widget class="QWidget" name="SearchBarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="CloseButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Close search bar</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources/resources.qrc">
       <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
     <property name="buddy">
      <cstring>TextField</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="TextField"/>
   </item>
   <item>
    <widget class="QToolButton" name="FindNextButton">
     <property name="toolTip">
      <string>Find next occurence of a string</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
     <property name="icon">
      <iconset resource="Resources/resources.qrc">
       <normaloff>:/icons/arrow_down.png</normaloff>:/icons/arrow_down.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="FindPrevButton">
     <property name="toolTip">
      <string>Find previous occurence of a string</string>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
     <property name="icon">
      <iconset resource="Resources/resources.qrc">
       <normaloff>:/icons/arrow_up.png</normaloff>:/icons/arrow_up.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="CaseCheckbox">
     <property name="text">
      <string>Case Sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="InfoLabel">
     <property name="text">
      <string>InfoText</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>182</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TextField</tabstop>
  <tabstop>FindNextButton</tabstop>
  <tabstop>FindPrevButton</tabstop>
  <tabstop>CaseCheckbox</tabstop>
  <tabstop>CloseButton</tabstop>
 </tabstops>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
